data/json/items/comestibles/raw_veggy.json

```diff
@@ -626,5 +626,47 @@
     "material": [ "veggy" ],
     "volume": 1,
     "price": 0
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "raw_beans",
+    "name": "raw beans",
+    "name_plural": "raw beans",
+    "color": "brown",
+    "spoils_in": "2 days",
```

Comment:
Only two days? I thought they could last for much longer.

Reply:
This is based on the shelf life for [green beans](https://www.eatbydate.com/proteins/beans-peas/beans-shelf-life-expiration-date/). Dried beans, which exist in game, last for much longer, and are probably what you are thinking of.

Reply:
Oh, I got it.